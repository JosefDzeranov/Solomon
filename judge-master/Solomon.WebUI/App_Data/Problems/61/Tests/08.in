60 12
ammo
ammo
shoot
shoot
ammo
shoot
shoot
ammo
shoot
shoot
ammo
ammo
shoot
ammo
ammo
ammo
shoot
shoot
ammo
ammo
ammo
ammo
shoot
ammo
shoot
ammo
shoot
ammo
ammo
shoot
shoot
ammo
shoot
ammo
ammo
ammo
shoot
shoot
ammo
shoot
shoot
ammo
ammo
ammo
shoot
ammo
ammo
shoot
shoot
shoot
shoot
shoot
shoot
ammo
ammo
ammo
ammo
shoot
ammo
shoot
