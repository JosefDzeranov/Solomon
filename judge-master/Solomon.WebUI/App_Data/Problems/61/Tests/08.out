12
24
23
22
34
33
32
44
43
42
54
66
65
77
89
101
100
99
111
123
135
147
146
158
157
169
168
180
192
191
190
202
201
213
225
237
236
235
247
246
245
257
269
281
280
292
304
303
302
301
300
299
298
310
322
334
346
345
357
356
